### DISCUSSION TITLE
Any way to disable history on opening tabby?

### DISCUSSION THREAD

**Discussion by stickben** [Eugeny/tabby#6929] [Q&A]
I see my serial history ("History Restored") every time I open tabby, it can be confusing from session to session.  Can I disable history?

**Comment by aaaaaadrian** [comment]
I was just searching for this and didn't see an answer, but luckily I was able to find it in the app.

At least in version 1.0.200 on MacOS, if you go to Settings > Terminal > Scroll to the bottom and there is a toggle for "Restore terminal tabs on app start"
![CleanShot 2023-09-20 at 12 56 15](https://github.com/Eugeny/tabby/assets/17524696/01864df7-83aa-49d9-a4a4-7021d6b487cc)

**Comment by dxps** [comment]
I really like that _restore_ feature, but I'd prefer to no see that _History restored_ text.
Just started reevaluating this app. It seems great for now.

Any option to prevent that text to be shown, please?
Thanks

**Comment by ThatsMeGIT** [comment]
yes you can set show recent profiles in selector to 0
<img width="883" height="431" alt="image" src="https://github.com/user-attachments/assets/9c1ed1e7-fe0d-490f-aa85-5592234f3aec" />
